src/enr/enr.ts

```diff
@@ -0,0 +1,48 @@
+import assert = require("assert");
```

Comment:
from instead of require

Reply:
I don't think you have assert declared as dependency (it works in tests because it uses mocha magic assert)